platform/uwb-beacon/platform.c

```diff
@@ -64,6 +65,27 @@ void fault_handler(void)
     reboot_system(BOOT_ARG_START_BOOTLOADER_NO_TIMEOUT);
 }
 
+static void delay(int n)
+{
+    for (volatile int i = 0; i < n; i++) {
+        asm volatile ("nop");
+    }
+}
+
+static void reboot_blink(void)
+{
+    const int period = 700 * 1000;
```

Comment:
How did you get 700 ?

Reply:
By trial and error until I found a period that looked good on the board :sweat: no hard science here.